x/leverage/client/tests/tests.go

```diff
@@ -386,7 +377,7 @@ func (s *IntegrationTestSuite) TestLeverageScenario() {
 		cli.GetCmdRepay(),
 		[]string{
 			val.Address.String(),
-			"51uumee",
+			"50uumee",
```

Comment:
Changes in rounding behavior have created a number of simpler decimals in this test file.